File: sdkwork-app-sdk-csharp/Models/PrivacySettings.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class PrivacySettings
    {
        public bool? PublicProfile { get; set; }
        public bool? AllowSearch { get; set; }
        public bool? AllowFriendRequest { get; set; }
        public bool? AllowStrangerMessage { get; set; }
        public bool? LocationShare { get; set; }
    }
}

File: sdkwork-app-sdk-typescript/src/types/privacy-settings.ts
/** 隐私设置 */
export interface PrivacySettings {
  /** 公开资料 */
  publicProfile?: boolean;
  /** 允许搜索 */
  allowSearch?: boolean;
  /** 允许好友请求 */
  allowFriendRequest?: boolean;
  /** 允许陌生人消息 */
  allowStrangerMessage?: boolean;
  /** 位置共享 */
  locationShare?: boolean;
}
